osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableDrumRoll.cs

```diff
@@ -56,6 +56,8 @@ public DrawableDrumRoll(DrumRoll drumRoll)
 
         protected override TaikoPiece CreateMainPiece() => new ElongatedCirclePiece();
 
+        public override bool OnPressed(TaikoAction action) => false;

```

Comment:
the base class' implenentation is abstract for `OnPressed` (i figured it was best to ensure each hit type was handling down, but as taiko never cares about up keys i added that as virtual to the bass class)